### PR TITLE
Deny cargo clippy lints

### PR DESCRIPTION
**Pull request by bgurney-rh**
Closes: #2408

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -4,6 +4,11 @@ version = "2.4.0"
 authors = ["Stratis Developers <stratis-devel@lists.fedorahosted.com>"]
 edition = "2018"
 build = "build.rs"
+description = "Stratis daemon"
+license = "MPL-2.0"
+repository = "https://github.com/stratis-storage/stratisd/"
+keywords = ["storage", "filesystem"]
```
Comment (mulkieran) [inline Cargo.toml:10]: We get a maximum of 5 keywords. We should definitely use "linux", also. You might want to look for likely words on crates.io.

**Review by mulkieran — CHANGES_REQUESTED** [review]
One request, otherwise looks good.

File: Cargo.toml (hunk 1 above)
Comment (bgurney-rh) [reply Cargo.toml:10]: The Cargo Book states that "you may choose any words that would help someone find this crate.  (Ah, the wonderful world of Search Engine Optimization; thankfully, it's limited to five words.)

Compare to the "categories" field, which must have strings exactly matching the category strings in https://crates.io/category_slugs (which, unfortunately, doesn't seem to have any storage-related categories, beyond "filesystem".)

**Review by mulkieran — CHANGES_REQUESTED** [review]
Looks good! Please close and open a final PR.

**Pull request closed without merging**